packages/automerge-repo/test/EphemeralData.test.ts

```diff
@@ -35,10 +35,14 @@ describe("EphemeralData", () => {
         done(e)
       }
     })
-    ephemeral.receive(
-      otherPeerId,
-      ("m/" + destinationChannelId) as ChannelId, // TODO: this is nonsense
-      CBOR.encode(messageData)
-    )
+    ephemeral.receive({
+      targetId: targetPeerId,
+      senderId: otherPeerId,
+      channelId: destinationChannelId,
```

Comment:
should we do away with channelIds? there's a 1:1 mapping of ephemeral messages to documents right now so no reason to use a different name here.

Reply:
I don't think there is any mapping of ephemeral messages to documents right now, is there? I thought you could give it whatever channelId you wanted, as it doesn't pass through the synchronizer?